nuget/readme.txt

```diff
@@ -0,0 +1,24 @@
+-------------------------------------------
+App Center SDK for .NET Readme
+-------------------------------------------
+
+Follow the “Getting Started” link to set up your application with App Center SDK. Then click on the service link for each App Center service that you want to enable in your app.
+
+-----Getting Started-----
+Xamarin: https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/xamarin
```

Comment:
blank lines after the header should be better

Reply:
Not sure what should be changed here?

Reply:
blank lines after each header section